### ISSUE TITLE
[node_7.x] Debian stretch "not currently supported" 

### ISSUE THREAD

**Issue by curlyel** [nodesource/distributions#493]
When installing using

`curl -sL https://deb.nodesource.com/setup_7.x | bash -`

The script reports "stretch" not being supported:

## Populating apt-get cache...

+ apt-get update
Hit:1 http://security.debian.org/debian-security stretch/updates InRelease
Ign:2 http://ftp.de.debian.org/debian stretch InRelease
Hit:4 http://ftp.de.debian.org/debian stretch-updates InRelease
Hit:5 http://ftp.de.debian.org/debian stretch Release
Reading package lists... Done

## Confirming "stretch" is supported...

+ curl -sLf -o /dev/null 'https://deb.nodesource.com/node_7.x/dists/stretch/Release'

## Your distribution, identified as "stretch", is not currently supported, please contact NodeSource at https://github.com/nodesource/distributions/issues if you think this is incorrect or would like your distribution to be considered for support

**Comment by chrislea** [comment]
The problem is that you're trying to install 7.x, which is already EOL. If you use the setup script for 8.x you'll find Stretch is supported.

**Comment by timbaileyjones** [comment]
I was having the same trouble, but I figured out that I was using a Vagrant box that did not have the ca-certificate package installed.  For that reason, curl was refusing to connect. Then I added -k to the curl command (ignore cert problems).  Then the downloaded script was ALSO invoking curl (but without -k), and as a result of that, the 2nd curl was returning a non-zero, which caused the script to incorrectly tell me that stretch was not supported.

So, after `apt-get install ca-certificates`, everything's fine.
